app/pulp/app/models/progress.py

```diff
@@ -181,15 +181,6 @@ class ProgressBar(ProgressReport):
         >>> with progress_bar:
         >>>     for file in progress_bar.iter(files_iterator):
         >>>         handle(file)
-
-    You can also use this short form:
-
-        >>> with ProgressBar(message='Publishing files', total=len(files_iterator)):
```

Comment:
Oh you are right. Instead of deleting it, what about updating it to be this:

`with ProgressBar.objects.create(message='Publishing files', total=len(files_iterator)) as progress_bar:`